src/utils/hact/workOrderSchedule/raiseWorkOrderForm.js

```diff
@@ -19,7 +19,7 @@ export const buildScheduleWorkOrderFormData = (workOrderData) => {
         lowPriority: !HIGH_PRIORITY_CODES.includes(
           Number.parseInt(workOrderData.priorityCode)
         ),
-        plannedPriority: Number.parseInt(workOrderData.priorityCode) == 9,
+        plannedPriority: Number.parseInt(workOrderData.priorityCode) === 9,
```

Comment:
We need to use `PLANNED_PRIORITY_CODE` here

Reply:
ah, sorry for that